style.css

```diff
@@ -0,0 +1,922 @@
+    place-items: center;
+    background-image: url("images/abstract-background-lineart-cubes_95169-324.jpg");
+    background-size: cover;
+    height: 490px;
+    padding: 1rem 3rem 1rem 3rem;
+  }
+
+  .main h2 {
+    color: #fff;
+    font-family: var(--font-family);
+    font-size: 27px;
+  }
+
+  .line {
+    background-color: #ec5242;
+    width: 3rem;
+    height: 2px;
+    margin-top: -6rem;
+  }
+
+  .big-box {
+    display: flex;
+    font-family: var(--font-family);
+    margin-top: -3rem;
+  }
+
+  .big-box .inner-box {
+    margin: 3px;
+    background-color: rgba(247, 246, 243, 0.2);
+    width: 10rem;
+    padding: 0.6rem;
+  }
+
+  .big-box .inner-box:hover {
+    border: #fff 2px solid;
+  }
+
+  .end-box strong img {
+    height: 2.6rem;
+    margin-top: 0.5rem;
+  }
+
+  .partner ul li img {
+    height: 3rem;
+    margin: 0 1rem 0 1rem;
+  }
+
+  .big-box .inner-box img {
+    margin-left: 3rem;
+  }
+
+  .big-box .inner-box h4 {
+    color: #ec5242;
+    font-size: 20px;
+    text-align: center;
+    margin-top: -0.1rem;
+  }
+
+  .main p {
+    color: #fff;
+    font-size: 18px;
+    font-family: var(--font-family);
+    margin-top: -1rem;
+  }
+
+  .main .underline:hover {
+    text-decoration: underline;
+  }
+
+  .feature p {
+    color: #000;
+    font-family: var(--font-family);
+    font-size: 14px;
+  }
+
+  .about-top p {
+    font-family: var(--font-family);
+    font-size: 16px;
+    color: #7a7a7a;
+    text-align: center;
+    margin-top: -5rem;
+  }
+
+  .second p {
+    color: #838383;
+    font-size: 16px;
+    font-family: var(--font-family);
+    width: 45vw;
+    text-align: center;
+    margin-top: -3rem;
+  }
+
+  .red p {
+    color: #838383;
+    font-size: 15px;
+    font-family: var(--font-family);
+  }
+
+  .big-box .inner-box p {
+    font-size: 12px;
+    text-align: center;
+    color: #fff;
+    margin-top: -0.5rem;
+  }
+
+  .speakers {
+    display: grid;
+    place-items: center;
+  }
+
+  .speakers h1 {
+    color: #000;
+    font-family: var(--font-family);
+    font-size: 27px;
+  }
+
+  .feature {
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    grid-template-rows: repeat(3, 15rem);
+    margin-top: 2rem;
+  }
+
+  .feature h2 {
+    color: #000;
+    font-family: var(--font-family);
+    font-size: 24px;
+  }
+
+  .orange {
+    color: #f78c89;
+    font-family: var(--font-family);
+    font-size: 15px;
+  }
+
+  .j-grid {
+    display: flex;
+  }
+
+  .div-g {
+    margin-left: 2rem;
+    width: 19rem;
+  }
+
+  .partner {
+    display: grid;
+    place-items: center;
+    height: 190px;
+    padding: 1.5rem 3rem 1rem 3rem;
+    background-color: #414246;
+  }
+
+  .partner h2 {
+    font-family: var(--font-family);
+    color: #8f8f91;
+    font-size: 28px;
+    margin-bottom: 1rem;
+  }
+
+  .l-2 {
+    margin-top: -1rem;
+  }
+
+  .partner ul {
+    display: flex;
+  }
+
+  footer {
+    padding: 3rem 3rem 3rem 6rem;
+    height: 6rem;
+    display: flex;
+  }
```

Comment:
- [ ]Now let's put our focus on the footer with the same momentum. We need to ensure that we stick to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. This will enable us to make the website more professional and user-friendly.  Please I kindly suggest centering the contents of the footer so as to match the original design ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226078316-538370f4-1796-4cbf-a3ec-2d6a4f031bfe.png)

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/226078336-6a978fd7-d385-458f-a040-674d2b1e3094.png)